packages/types/src/order/workflows.ts

```diff
@@ -0,0 +1,8 @@
+import { CustomerDTO } from "../customer"
+import { RegionDTO } from "../region"
+import { OrderDTO } from "./common"
+
+export interface OrderWorkflowDTO extends OrderDTO {
+  customer?: CustomerDTO
+  region?: RegionDTO
```

Comment:
Q: goes with some other about typings, do we expect the entire region and customer? I believe that worflows are in a separate layer, so we can have differences in the DTO to specify what is needed by the workflows. 

It is a more global discussion no need to change it for nw 
cc @olivermrbl